verilog/preprocessor/verilog_preprocess.cc

```diff
@@ -212,6 +232,19 @@ void VerilogPreprocess::RegisterMacroDefinition(
   // TODO(hzeller): multiline warning with 'previously defined here' location
 }
 
+absl::Status VerilogPreprocess::ExpandMacro(MacroDefinition& definition){
+  VerilogLexer lexer(definition.DefinitionText().text());
+  verible::TokenSequence myseq;
+  while (true) {
```

Comment:
this `while(true)` ... `DoNextToken()` ... `break` loop sounds like it can be formulated as `for`-loop.